data/sql/updates/pending_db_world/get-model-for-player.sql

```diff
@@ -0,0 +1,173 @@
+DROP TABLE IF EXISTS `player_shapeshift_model`;
+DROP TABLE IF EXISTS `player_totem_model`;
+
+CREATE TABLE IF NOT EXISTS `player_shapeshift_model` (
+  `ShapeshiftID` TINYINT unsigned NOT NULL,
+  `RaceID` TINYINT unsigned NOT NULL,
+  `CustomizationID` TINYINT unsigned NOT NULL,
+  `GenderID` TINYINT unsigned NOT NULL,
+  `ModelID` INT unsigned NOT NULL,
+  PRIMARY KEY (`ShapeshiftID`, `RaceID`, `CustomizationID`, `GenderID`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 PACK_KEYS=0;
```

Comment:
In addition, there should be someone requesting the use of 'InnoDB' here.

Reply:
You're probably right, I just used MyISAM here since it's what I used last time I added a table, and had no opposition.

Reply:
Corrected.

Reply:
May be worth adding that to the SQL codestyle check at some point too, just for consistency.